### ISSUE TITLE
could not install

### ISSUE THREAD

**Issue by rlucian** [rkit/filemanager-yii2#1]
```
Using version ^1.0 for rkit/filemanager-yii2
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - rkit/filemanager-yii2 1.0.0 requires intervention/image ^2.3.2 -> satisfiable by intervention/image[2.3.2, 2.3.3, 2.3.4].
    - rkit/filemanager-yii2 1.0.1 requires intervention/image ^2.3.2 -> satisfiable by intervention/image[2.3.2, 2.3.3, 2.3.4].
    - rkit/filemanager-yii2 1.0.2 requires intervention/image ^2.3.2 -> satisfiable by intervention/image[2.3.2, 2.3.3, 2.3.4].
    - rkit/filemanager-yii2 1.0.3 requires intervention/image ^2.3.2 -> satisfiable by intervention/image[2.3.2, 2.3.3, 2.3.4].
    - intervention/image 2.3.4 requires ext-fileinfo * -> the requested PHP extension fileinfo is missing from your system.
    - intervention/image 2.3.3 requires ext-fileinfo * -> the requested PHP extension fileinfo is missing from your system.
    - intervention/image 2.3.2 requires ext-fileinfo * -> the requested PHP extension fileinfo is missing from your system.
    - Installation request for rkit/filemanager-yii2 ^1.0 -> satisfiable by rkit/filemanager-yii2[1.0.0, 1.0.1, 1.0.2, 1.0.3].


Installation failed, reverting ./composer.json to its original content.
```

I do have fileinfo php extension installed.

```
fileinfo

fileinfo support    enabled
version 1.0.5
```

the workarround was 
composer install --ignore-platform-reqs rkit/filemanager-yii2

**Comment by rkit** [comment]
Maybe your composer running from another php?
—
I released a new version (1.0.4)
Moved `intervention/image` in section `require-dev`.
Because you can use any library for working with files.

**Comment by rkit** [comment]
I closed. If you have any problems, please, let me know it…
